### DISCUSSION TITLE
FR: Alerting Contact Point Matrix

### DISCUSSION THREAD

**Discussion by Chaostheorie** [grafana/grafana#65352] [Alerting]
Hi, thank you for your work on Grafana. The alerting module is an awesome however I'd like to request/ add a new contact point for interfacing with [Matrix](https://matrix.org/) servers.

One of the reasons for wanting this in addition to the existing services is the ability to self-host the Matrix Server and as such, among other elements, isolate notifications from third-parties. There are other advantages too, like the various security elements of the Matrix protocol or the ability to create more contact points on a self-hosted server compared to some hosted/ SaaS alternatives.

This would, regarding implementation, look similar to the Telegram Contact Point. Though a client library, like [mautrix](https://github.com/mautrix/go), will most likely be required.

If this feature aligns with the plans for Grafana Alerting, then I would try to contribute it myself.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82764. Feel free to continue the discussion around this there. Thank you!
